toothpick-runtime/build.gradle

```diff
@@ -10,22 +8,23 @@ configurations {
 }
 
 dependencies {
-  api project(':toothpick-generated-core')
+  api project(':toothpick')
   api deps.inject
 
-  testImplementation deps.junit
-  testImplementation project(':toothpick-generated-core')
+  testImplementation deps.junit4
   testImplementation deps.easymock
   testImplementation deps.powermock
   testImplementation deps.hamcrest
-  testImplementation files(Jvm.current().getToolsJar())
+  testImplementation 'org.powermock:powermock-module-junit4-rule-agent:1.6.4'
   testAnnotationProcessor project(':toothpick-compiler')
 }
 
+test {
+  jvmArgs '-noverify'
+}
```

Comment:
should we keep this?

Reply:
yes, powermock breaks the java 7 byte code frames otherwise.